.gitignore

```diff
@@ -1,6 +1,7 @@
 bower_components/
 node_modules/
 
+package-lock.json
```

Comment:
So as per NPM documents, this is intended to be included in the repo

Reply:
Have you decided what you'd like to do here?

Reply:
Reverted on PR https://github.com/nightscout/cgm-remote-monitor/pull/2991 . `package-lock.json` is removed from dev branch and we now use npm-shrinkwrap.json
So no need for .gitignore.